python/ray/scripts/symmetric_run.py

```diff
@@ -0,0 +1,382 @@
+    if node_manager_port:
+        ray_start_cmd.extend(["--node-manager-port", str(node_manager_port)])
+    if object_manager_port:
+        ray_start_cmd.extend(["--object-manager-port", str(object_manager_port)])
+    if runtime_env_agent_port:
+        ray_start_cmd.extend(["--runtime-env-agent-port", str(runtime_env_agent_port)])
+    if dashboard_agent_grpc_port:
+        ray_start_cmd.extend(
+            ["--dashboard-agent-grpc-port", str(dashboard_agent_grpc_port)]
+        )
+    if dashboard_agent_listen_port:
+        ray_start_cmd.extend(
+            ["--dashboard-agent-listen-port", str(dashboard_agent_listen_port)]
+        )
+    if metrics_export_port:
+        ray_start_cmd.extend(["--metrics-export-port", str(metrics_export_port)])
+    return ray_start_cmd
+
+
+@click.command()
+@click.option(
+    "--address", required=True, type=str, help="The address of the Ray cluster."
+)
+@click.option(
+    "--num-cpus",
+    type=int,
+    help="The number of CPUs to use for the Ray cluster. If not provided, "
+    "the number of CPUs will be automatically detected.",
+)
+@click.option(
+    "--num-gpus",
+    type=int,
+    help="The number of GPUs to use for the Ray cluster. If not provided, "
+    "the number of GPUs will be automatically detected.",
+)
+@click.option(
+    "--wait-for-nnodes",
+    type=int,
+    help="If provided, wait for this number of nodes to start.",
+)
+@click.option(
+    "--disable-usage-stats",
+    is_flag=True,
+    default=False,
+    help="If True, the usage stats collection will be disabled.",
+)
+@click.option(
+    "--resources",
+    required=False,
+    default="{}",
+    type=str,
+    help="A JSON serialized dictionary mapping resource name to resource quantity.",
+)
+@click.option(
+    "--min-worker-port",
+    type=int,
+    help="the lowest port number that workers will bind on. If not set, "
+    "random ports will be chosen.",
+)
+@click.option(
+    "--max-worker-port",
+    type=int,
+    help="the highest port number that workers will bind on. If set, "
+    "'--min-worker-port' must also be set.",
+)
+@click.option(
+    "--node-manager-port",
+    type=int,
+    default=0,
+    help="the port to use for starting the node manager",
+)
+@click.option(
+    "--object-manager-port",
+    type=int,
+    help="the port to use for starting the object manager",
+)
+@click.option(
+    "--runtime-env-agent-port",
+    type=int,
+    help="The port for the runtime environment agents to listen for http on.",
+)
+@click.option(
+    "--dashboard-agent-grpc-port",
+    type=int,
+    help="the port for dashboard agents to listen for grpc on.",
+)
+@click.option(
+    "--dashboard-agent-listen-port",
+    type=int,
+    help="the port for dashboard agents to listen for http on.",
+)
+@click.option(
+    "--metrics-export-port",
+    type=int,
+    default=None,
+    help="the port to use to expose Ray metrics through a Prometheus endpoint.",
+)
```

Comment:
Try use

```
@click.command(context_settings=dict(
    ignore_unknown_options=True,
    allow_extra_args=True
))
```

for pass-through options to `ray start` instead of defining them repeatedly here.

Reply:
But we can't do that I think since we need the entrypoint args to also be passed through